packages/expo-sqlite/ios/CRSQLiteLoader.h

```diff
@@ -0,0 +1,5 @@
+// Copyright 2015-present 650 Industries. All rights reserved.
+
+#pragma once
+
+int crsqlite_init_from_swift(void);
```

Comment:
It will have to be in `expo` namespace, otherwise we will get duplicated symbols after versioning